unit_testing/cli/test_cli_sct_dice_coefficient.py

```diff
@@ -1,14 +1,21 @@
-from pytest_console_scripts import script_runner
 import pytest
 import logging
 
+from spinalcordtoolbox.image import Image, compute_dice
+from spinalcordtoolbox.scripts import sct_dice_coefficient
+
 logger = logging.getLogger(__name__)
 
 
-@pytest.mark.script_launch_mode('subprocess')
-def test_sct_dice_coefficient_backwards_compat(script_runner):
-    ret = script_runner.run('sct_testing', '--function', 'sct_dice_coefficient')
-    logger.debug(f"{ret.stdout}")
-    logger.debug(f"{ret.stderr}")
-    assert ret.success
-    assert ret.stderr == ''
+@pytest.mark.sct_testing
+@pytest.mark.usefixtures("run_in_sct_testing_data_dir")
+def test_sct_dice_coefficient_check_output_against_groundtruth():
+    """Run the CLI script and verify its output matches a known ground truth value."""
+    # FIXME: The results of "sct_dice_coefficient" are not actually verified. Instead, the "compute_dice" function
+    #        is called, and THOSE results are verified instead.
+    # This was copied as-is from the existing 'sct_testing' test, but should be fixed at a later date.
+    path_data = 't2/t2_seg-manual.nii.gz'
+    sct_dice_coefficient.main(argv=['-i', path_data, '-d', path_data])
+    im_seg_manual = Image(path_data)
+    dice_segmentation = compute_dice(im_seg_manual, im_seg_manual, mode='3d', zboundaries=False)
+    assert dice_segmentation == 1.0
```

Comment:
what about the assertion message like the original test `'\nERROR: Dice coefficient should be : ' + str(param_test.dice_value)`

Reply:
Here is what an example failure would look like (with no assertion message):

```
unit_testing/cli/test_cli_sct_dice_coefficient.py::test_sct_dice_coefficient_check_output_against_groundtruth FAILED [100%]

============================================== FAILURES ==============================================
_____________________ test_sct_dice_coefficient_check_output_against_groundtruth _____________________
Traceback (most recent call last):
  File "/home/joshua/repos/spinalcordtoolbox/unit_testing/cli/test_cli_sct_dice_coefficient.py", line 22, in test_sct_dice_coefficient_check_output_against_groundtruth
    assert dice_segmentation == 1.0
AssertionError: assert 2.0 == 1.0
  +2.0
  -1.0
====================================== short test summary info =======================================
FAILED unit_testing/cli/test_cli_sct_dice_coefficient.py::test_sct_dice_coefficient_check_output_against_groundtruth
========================================= 1 failed in 2.90s ==========================================
```

I was thinking that the pytest output might be clear enough that we don't need an additional message.